ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -662,12 +664,18 @@ extension ReactIterableAPI: IterableAuthDelegate {
                 DispatchQueue.main.async {
                     completion(self.passedAuthToken)
                 }
+                
+                self.sendEvent(withName: EventName.handleAuthSuccessCalled.rawValue,
```

Comment:
Do we need .rawValue here or would it automatically map to correctly without it?

Reply:
Just keeping it consistent with other Send events as of now. 
Send events usually take strings and .rawValue can extract that from enum.